opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -23,25 +23,37 @@
     /**
      * Return a new SpanBuilder for a Span with the given `operationName`.
      *
+     * <p>If there is an active Span according to the {@link Tracer#scheduler()}'s
```

Comment:
This semantic change is the one (huge) benefit afforded by all of the additional complexity in this PR. (I added this comment before, but reiterating since it got lost in the https://github.com/opentracing/opentracing-java/pull/112 rebase)